docs/source/intervention_models/multiple_myeloma_treatment/index.rst

```diff
@@ -298,98 +298,316 @@ We plan to use Flatiron to inform baseline coverage of each treatment regimen.
+In Phase 2, we are expanding the treatment regimen categories considerably, to a
+total of 24, and making transitions dependent on prior treatment and covariates.
 
-Treatment Regimen Categories
-++++++++++++++++++++++++++++
+Treatment Regimen Category Components
++++++++++++++++++++++++++++++++++++++
 
-The Phase 1 simulation only considered three categories of treatment regimen:
-isatuximab-containing, daratumumab-containing, and other. Based on conversations
-with the client and with our clinical expert Manoj Menon, we plan to expand the
-modeled treatment categories to the following set of 16 mutually exclusive
-categories:
+These are the components that make up our modelled treatment regimen categories.
+They are drug classes, individual drugs, and ASCT.
 
-.. list-table:: Modeled Treatment Regimen Categories
-  :widths: 5 10 8 15
+.. note::
+
+  Not every combination of these components is a valid treatment regimen category; those
+  are enumerated in the next section.
+
+.. list-table:: Modeled Treatment Regimen Category Components
+  :widths: 1 1 5 10
   :header-rows: 1
 
-  * - Enumeration
-    - Treatment category
-    - Supercategory
+  * - Drug class
+    - Abbreviation
+    - Drugs
     - Notes
-  * - 1
-    - PI/Dex
-    -
-    -
-  * - 2
-    - IMID/Dex
-    -
-    -
-  * - 3
-    - PI/IMID/Dex
-    -
-    -
-  * - 4
-    - Chemo/PI/Dex
+  * - Proteasome inhibitors
+    - PI
+    - bortezomib, carfilzomib, ixazomib
     -
+  * - Immunomodulatory drugs
+    - IMID
+    - lenalidomide, pomalidomide, thalidomide
     -
-  * - 5
-    - Chemo/IMID/Dex
+  * - Chemotherapy drugs
+    - Chemo
+    - bendamustine, cyclophosphamide, doxorubicin
     -
+  * - Daratumumab
+    - Dara
+    - daratumumab
     -
-  * - 6
-    - Dara/bortezomib/Dex
-    - Dara/PI/Dex
-    -
-  * - 7
-    - Dara/carfilzomib/Dex
-    - Dara/PI/Dex
-    -
-  * - 8
-    - Dara/ixazomib/Dex
-    - Dara/PI/Dex
-    -
-  * - 9
-    - Dara/lenalidomide/Dex
-    - Dara/IMID/Dex
-    -
-  * - 10
-    - Dara/pomalidomide/Dex
-    - Dara/IMID/Dex
+  * - Isatuximab
+    - Isa
+    - isatuximab
     -
-  * - 11
-    - Dara/thalidomide/Dex
-    - Dara/IMID/Dex
+  * - Autologous stem cell transplant
+    - ASCT
+    - N/A
+    - While ASCT is a subsequent step after induction therapy, we consider it
+      part of the same line and it is included in the treatment regimen category as if it
+      were at the time of induction. We restrict it to only appear in NDMM (while later transplants
+      do occur in real life, they are relatively uncommon, and we do not model them.)
+
+.. note::
+
+  The drug class components represent *exactly one* drug from
+  that class. If there is more than one, the treatment regimen will always fall
+  into one of the Other treatment regimen categories described in the next section.
+
+Treatment Regimen Categories
+++++++++++++++++++++++++++++
+
+Based on conversations
+with the client and with our clinical expert Manoj Menon, we are expanding the
```

Comment:
We've never explicitly named anyone in these public docs before (at least to my knowledge) - does anyone have any thoughts on whether we should call out Manoj here or leave as simply 'our clinical expert'?

Reply:
For some reason, my gut reaction is to avoid naming the CSU clients, but no problem with naming our expert advisors.  Let's discuss briefly and decide about our approach together.  If Zeb wants to merge before we talk, please do not include the name for now.